text/0985-v2-addon-by-default.md

```diff
@@ -0,0 +1,81 @@
+---
+stage: accepted
+start-date: 2023-11-02T16:05:11.000Z 
+release-date: # In format YYYY-MM-DDT00:00:00.000Z
+release-versions:
+teams: # delete teams that aren't relevant
+  - cli
+  - data
+  - framework
+  - learning
+  - steering
+  - typescript
+prs:
+  accepted: https://github.com/emberjs/rfcs/pull/985
+project-link:
+suite: 
+---
+
+<!--- 
+Directions for above: 
+
+stage: Leave as is
+start-date: Fill in with today's date, 2032-12-01T00:00:00.000Z
+release-date: Leave as is
+release-versions: Leave as is
+teams: Include only the [team(s)](README.md#relevant-teams) for which this RFC applies
+prs:
+  accepted: Fill this in with the URL for the Proposal RFC PR
+project-link: Leave as is
+suite: Leave as is
+-->
+
+# Change the default addon blueprint
+
+## Summary
+
+`@embroider/addon-blueprint` has been in progress for a while, it's good stuff. 
+
+
+## Motivation
+
+We want to encourage folks to use v2 addons instead of the classic addon blueprint.
+
+V1 addons are built by apps on every boot, every build.
+V2 addons are built at publish time, so the app can build faster.
+
+## Detailed design
+
+`@embroider/addon-blueprint` will remain its own repo. `ember addon` will use this by default without a flag at some point.
```

Comment:
of note, ember-cli 5.4 broke the `@embroider/addon-blueprint`.

So, that's the biggest risk with using an external blueprint with no other changes to process.

The details of flags of how blueprints are created (`packageManager` in this case vs `yarn` or `npm` flags) don't feel exactly like public API, so I'm sure that makes this more complicated.